docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -63,6 +63,44 @@ status: {}
 More information about the [scale subresource can be found in the Kubernetes
 documentation][scale].
 
+#### Required `spec` fields for implementations using version
+
+* `version` - is a string representing the Kubernetes version to be used
+  by the control plane machines. The value must be a valid semantic version;
+  also if the value provided by the user does not start with the v prefix, it
+  should be added.
+  
+#### Required `status` fields for implementations using version
```

Comment:
can this move down with the rest of the status field requirements? status for implementations using replicas is at the bottom after all the spec requirements